src/main/java/frc/robot/lib/interfaces/Arm.java

```diff
@@ -75,31 +79,43 @@ public Arm() {
         RobotMap.leftElbowMotor.configReverseSoftLimitThreshold(Constants.ELBOW_MOTOR.B_TRAVEL_LIMIT);
         RobotMap.rightElbowMotor.configReverseSoftLimitThreshold(Constants.ELBOW_MOTOR.B_TRAVEL_LIMIT);
 
+        // TODO: config mount pose of IMU
+
         RobotMap.rightShoulderMotor.set(ControlMode.Follower, Constants.CAN_IDS.LEFT_SHOULDER_MOTOR);
         RobotMap.rightElbowMotor.set(ControlMode.Follower, Constants.CAN_IDS.LEFT_ELBOW_MOTOR);
     }
 
     public void setIdle() {
+        // TODO: IMU degree
         moveArmPosition(100.0, 0.0);
     }
 
     public void setElbowIdle() {
         RobotMap.leftShoulderMotor.getSelectedSensorPosition();
-        RobotMap.leftElbowMotor.set(TalonFXControlMode.Position, Constants.ELBOW_IDLE.ELBOW_POSITION);
+
+        // TODO: figure out if reading pitch or roll
+        elbowCommand = controller.calculate(RobotMap.elbowIMU.getRoll(), Constants.ELBOW_IDLE.ELBOW_POSITION);
+        RobotMap.leftElbowMotor.set(elbowCommand);
+        
         elbowTarget = Constants.ELBOW_IDLE.ELBOW_POSITION;
     }
 
+    // encoder position for the shoulder, IMU degree for the elbow
     public void moveArmPosition(double shoulder, double elbow) {
         RobotMap.leftShoulderMotor.set(TalonFXControlMode.Position, shoulder);
-        RobotMap.leftElbowMotor.set(TalonFXControlMode.Position, elbow);
+
+        // TODO: figure out if reading pitch or roll
+        elbowCommand = controller.calculate(RobotMap.elbowIMU.getRoll(), elbow);
+        RobotMap.leftElbowMotor.set(elbowCommand);
+
         shoulderTarget = shoulder;
         elbowTarget = elbow;
     }
 
     // method to check if arm has arrived at its position
-    public static Boolean getArrived(double allowance, double time) {
-        if (Math.abs(RobotMap.leftShoulderMotor.getSelectedSensorPosition() - shoulderTarget) <= Math.abs(allowance) && 
-            Math.abs(RobotMap.leftElbowMotor.getSelectedSensorPosition() - elbowTarget) <= Math.abs(allowance)) {
+    public static Boolean getArrived(double shoulderAllowance, double elbowAllowance, double time) {
+        if (Math.abs(RobotMap.leftShoulderMotor.getSelectedSensorPosition() - shoulderTarget) <= Math.abs(shoulderAllowance) && 
+            Math.abs(RobotMap.leftElbowMotor.getSelectedSensorPosition() - elbowTarget) <= Math.abs(elbowAllowance)) {
```

Comment:
We are not using the internal motor encoder for position anymore, we are using the gyro. The sensor reading call needs to reflect that.